openbas-model/src/main/java/io/openbas/database/model/InjectExpectation.java

```diff
@@ -117,6 +117,12 @@ public EXPECTATION_STATUS getResponse() {
   @NotNull
   private Double expectedScore;
 
+  @Setter
+  @Column(name = "inject_expiration_time")
+  @JsonProperty("inject_expiration_time")
+  @NotNull
+  private Long expirationTime;
```

Comment:
We need to add a default value here like 0 to avoid DB error.
When an inject already exists, the expectations are in the content and are not handle by the migration.